sdk/scheduler/src/main/java/com/mesosphere/sdk/dcos/Capabilities.java

```diff
@@ -95,6 +95,11 @@ public boolean supportsPreReservedResources() {
         return hasOrExceedsVersion(1, 10);
     }
 
+    public boolean supportsStrictModeV1API() {
```

Comment:
Nit:  technically, I believe it will work against 1.10 strict, so this should probably be called something like "useV1ByDefault".

Reply:
Agreed on method name